sources/histogram.py

```diff
@@ -2,5 +2,8 @@
 
 # Quel cours de Poudlard a une repartition des notes homogenes entre les quatres maisons ?
 
+# think about "mugging" ? google.ml.doc
```

Comment:
i don't watch this type of porn

Reply:
this is just to remind me of something, i can delete.